packages/apps/Settings/res/values-it/mad_strings.xml

```diff
@@ -935,5 +935,62 @@
 
     <!-- Quick unlock -->
     <string name="unlock_quick_unlock_control_title">Sblocco rapido</string>
-    <string name="unlock_quick_unlock_control_summary">Sblocca automaticamente non appena il PIN corretto è stato inserito (senza premere "OK")\nNote: funziona solo con PIN a 4 cifre!</string>
+    <string name="unlock_quick_unlock_control_summary">Sblocca automaticamente non appena il PIN corretto è stato inserito (senza premere "OK")\nNote: funziona solo se il numero sottostante di cifre è impostato correttamente!</string>
+    <string name="lockscreen_quick_unlock_pin_length_title">Numero di cifre del PIN</string>
+    <string name="lockscreen_quick_unlock_pin_length_summary">Per lo sblocco rapido, imposta la linghezza del PIN (numero di cifre).</string>
+
+    <!-- Immersive recents -->
+    <string name="immersive_recents_title">Modalità recenti immersiva</string>
+    <string name="immersive_recents_default">Default</string>
+    <string name="immersive_recents_fullscreen">A tutto schermo</string>
+    <string name="immersive_recents_statusbar">Solo barra di stato</string>
+    <string name="immersive_recents_navbar">Solo barra di navigazione</string>
+
+    <!-- Recent clear all button and location -->
+    <string name="show_clear_all_recents_button_title">Rimuovi tutto (FAB)</string>
```

Comment:
FAB = Floating Action Button

its the "Clear All Button" but in a custom style

Currently on our rom, we only have "Clear All Button", this adds another Custom Button "FAB" mostly the same as the one we had on Lollipop RC5